Task/Task-NonWGSTestOrderFormAccepted-UsingStoredSample-Example.json

```diff
@@ -0,0 +1,47 @@
+{
+    "resourceType": "Task",
+    "id": "Task-NonWGSTestOrderFormAccepted-UsingStoredSample-Example",
+    "basedOn": [
+        {
+            "reference": "ServiceRequest/ServiceRequest-NonWGSTestOrderForm-UsingStoredSample-Example"
+        }
+    ],
+    "status": "accepted",
+    "intent": "order",
+    "priority": "routine",
+    "code": {
+        "coding": [
+            {
+                "system": "https://fhir.nhs.uk/CodeSystem/task-code-genomics",
+                "code": "RequestGenomicTest",
+                "display": "Request Genomic Test"
+            }
+        ]
+    },
+    "focus": {
+        "reference": "ServiceRequest/ServiceRequest-NonWGSTestOrderForm-UsingStoredSample-Example"
+    },
+    "for": {
+        "reference": "Patient/Patient-TimMclullichs-Example",
+        "identifier": {
+            "system": "https://fhir.nhs.uk/Id/nhs-number",
+            "value": "8449303452"
+        }
+    },
+    "authoredOn": "2023-11-02T15:43:00Z",
+    "requester": {
+        "reference": "PractitionerRole/PractitionerRole-OscarShields-Example"
+    },
+    "owner": {
+        "identifier": {
+            "system": "https://fhir.nhs.uk/Id/ods-organization-code",
+            "value": "RQ3"
+        },
+        "display": "BIRMINGHAM WOMEN'S AND CHILDREN'S NHS FOUNDATION TRUST"
+    },
+    "note": [
+        {
+            "text": "Optional Free Text field to record any additional details"
```

Comment:
Should be "sample recieved and placed to be placed in storage"

Could also add output referencing Specimen resource to record the specimen stored